frontend/src/components/common/AccordionCard/AccordionCard.tsx

```diff
@@ -20,27 +21,29 @@ export const AccordionCard = ({
 
   return (
     <div className={className}>
-      <h2>
-        <button
-          type="button"
-          className={generateClassNames(
-            'flex w-full items-center justify-between rounded-2xl border border-navy-200 p-6',
-            {
-              'hover:bg-navy-100': !isOpen && children,
-              'hover:bg-transparent rounded-b-none border-b-0 pb-0': isOpen,
-              'cursor-auto': !children,
-            },
-          )}
-          onClick={() => setOpen(!isOpen)}
-        >
-          <span className={`${small ? 'text-base' : 'text-xl'} font-semibold text-navy-900`}>{title}</span>
-          {children ? (
-            <div className="flex h-8 w-8 items-center justify-center rounded-full bg-navy-50">
-              <ChevronRightIcon className={generateClassNames('rotate-90 text-navy-500', { '-rotate-90': isOpen })} />
-            </div>
-          ) : undefined}
-        </button>
-      </h2>
+      <button
+        type="button"
+        className={generateClassNames(
+          'flex w-full items-center justify-between rounded-2xl border border-navy-200 p-6',
+          {
+            'hover:bg-navy-100': !isOpen && children,
+            'hover:bg-transparent rounded-b-none border-b-0 pb-0': isOpen,
+            'cursor-auto': !children,
+          },
+        )}
+        onClick={() => setOpen(!isOpen)}
+      >
+        {small ? (
+          <Typography variant="body-m/semibold">{title}</Typography>
+        ) : (
+          <Typography variant="head-s/semibold">{title}</Typography>
+        )}
```

Comment:
maybe something like that:
```suggestion
          <Typography variant={small ? "body-m/semibold" : "head-s/semibold"}>{title}</Typography>
```